test/test_esapi.py

```diff
@@ -2843,6 +2843,17 @@ def test_GetSessionAuditDigest(self):
                 signHandle, session, b"1234", session3=12.723
             )
 
+    def test_PP_Commands(self):
+        with self.assertRaises(TSS2_Exception) as e:
+            self.ectx.PP_Commands(
+                ESYS_TR.PLATFORM, TPML_CC(), TPML_CC(), session1=ESYS_TR.PASSWORD
+            )
+        self.assertEqual(e.exception.error, TPM2_RC.PP)
+        self.assertEqual(e.exception.session, 1)
```

Comment:
negative test all inputs.

Reply:
do you mean "bad" inputs, such as wrong type?

Reply:
Yes

Reply:
Added some tests to check for TypeError when passing bad input